### DISCUSSION TITLE
Possible certificate error

### DISCUSSION THREAD

**Discussion by guyas** [SafeExamBrowser/seb-win-refactoring#799] [Q&A] [answered]
Hi
I'm trying to configure a SEB client that will go to an internal server which has an SSL cert which was produced internally as well. Normally, unless ignored, navigating with chrome to this address results with the following error:

![block](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/065380dc-6c9c-444a-92a0-7b5df0fcd0ad)

However when using the SEB (3.6.0) I get this:

![Capture](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/32ea572c-f639-4289-9af3-05a241190c0b)

Any ideas what to do?

**Comment by dbuechel** [comment]
At the moment, we do not have implemented any custom error pages for page load errors but only the simple message box you've posted above. The underlying error however is the same (ERR_CERT_COMMON_NAME_INVALID).

**Comment by guyas** [comment, marked as answer]
OK this was solved by locally installing the certificate. Thanks anyway!
